test/server.js

```diff
@@ -8,8 +8,10 @@ var MqttServer
 var MqttSecureServer
 
 function setupConnection (duplex) {
-  var connection = new Connection(duplex)
-  this.emit('client', connection)
+  var that = this
+  var connection = new Connection(duplex, function () {
+    that.emit('client', connection)
+  })
```

Comment:
why this change?

Reply:
this changed because logic of initialization of server was been changed. Server process packets by version in connect packet